ui/changes/src/common.js

```diff
@@ -140,53 +140,53 @@ export let landingsData = (async function () {
   return returnedObject;
 })();
 
-export function getNewTestingTagCountObject() {
-  let obj = {};
-  for (let tag in TESTING_TAGS) {
-    obj[tag] = 0;
+export class Counter {
+  constructor() {
+    return new Proxy(
+      {},
+      {
+        get: (target, name) => (name in target ? target[name] : 0)
+      }
+    );
   }
-  return obj;
 }
 
-export async function getTestingPolicySummaryData(grouping = "daily", filter) {
-  let data = await landingsData;
+export function getSummaryData(
+  bugSummaries,
+  grouping = "daily",
+  startDate,
+  filter,
```

Comment:
Nit: I'd probably flip the order of `filter` and `counter` since filter is unused by one of the consumers (so it can just be dropped from the parameters in that case instead of having to pass null)